packages/widget_toolkit/lib/src/lib_ui_components/buttons/small_button.dart

```diff
@@ -61,6 +61,15 @@ class SmallButton extends StatelessWidget {
       colorGradientEnd,
     ]);
 
+    final icon = state == ButtonStateModel.loading
```

Comment:
Please add golden tests for the states
- loading
- enabled
- disabled
- etc

Reply:
Done